docker-compose.yml

```diff
@@ -24,9 +24,22 @@ services:
       - tools
     networks:
       - recipe-app-network
+  cache:
+    image: redis:7.0
+    restart: always
+    ports:
+      - '6379:6379'
+    command: redis-server --save 20 1 --loglevel warning --requirepass '${REQUIRE_PASS_REDIS}'
```

Comment:
hmm, I don't think we need to persist the database for our purposes (remember this is cache only), but it can stay :)